XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml.cs

```diff
@@ -52,49 +56,49 @@ private void InitializeData()
             MixedTypeRepeater.ItemsSource = basicData;
 
             List<NestedCategory> nestedCategories = new List<NestedCategory>();
-            nestedCategories.Add(
-                new NestedCategory("Fruits",  new ObservableCollection<string>{
-                                                            "Apricots",
-                                                            "Bananas",
-                                                            "Grapes",
-                                                            "Strawberries",
-                                                            "Watermelon",
-                                                            "Plums",
-                                                            "Blueberries"
-                }));
-
-            nestedCategories.Add(
-                new NestedCategory("Vegetables", new ObservableCollection<string>{
-                                                            "Broccoli",
-                                                            "Spinach",
-                                                            "Sweet potato",
-                                                            "Cauliflower",
-                                                            "Onion",
-                                                            "Brussel sprouts",
-                                                            "Carrots"
-                }));
-
-            nestedCategories.Add(
-                new NestedCategory("Grains", new ObservableCollection<string>{
-                                                            "Rice",
-                                                            "Quinoa",
-                                                            "Pasta",
-                                                            "Bread",
-                                                            "Farro",
-                                                            "Oats",
-                                                            "Barley"
-                }));
-
-            nestedCategories.Add(
-                new NestedCategory("Proteins", new ObservableCollection<string>{
-                                                            "Steak",
-                                                            "Chicken",
-                                                            "Tofu",
-                                                            "Salmon",
-                                                            "Pork",
-                                                            "Chickpeas",
-                                                            "Eggs"
-                }));
+            ObservableCollection<string> fruits = new ObservableCollection<string>{
```

Comment:
you can use initialization syntax to make this look more like the nested collection structure in code instead of adding one by one.

Reply:
What do you mean exactly by adding one by one? I thought I was using the object initialization syntax with collection<T> foo = new collection<T> { item array }

Reply:
You can take it to both levels here like so.
var nestedCategories = new List<NestedCategory>()
{
   new NestedCategory("Vegitables", new ObservableCollection<string>(){"Broccoli", "Spinach"... },
   new NestedCategory("Fruits", new ObservableCollection<string>(){"Banana", "Grapes"... },

}